packages/data-context/src/actions/CodegenActions.ts

```diff
@@ -139,3 +150,70 @@ export class CodegenActions {
     return WIZARD_FRAMEWORKS.find((framework) => framework.configFramework === config?.component?.devServer.framework)
   }
 }
+
+type Linker = Record<string, { exportName: string, isDefault: boolean }>
```

Comment:
I haven't seen the term "linker" before - should this be called `resolver`? That's why the types call it.

Reply:
Addressed in edca01e60f1a06af1981f0956fe550f58644ec12!